test/utilities/constants.ts

```diff
@@ -17,3 +17,13 @@ export const WORKSPACE_SETTING_KEYS = {
 export const LOG_LEVELS = ['trace', 'debug', 'info', 'warn', 'error', 'silent'] as const;
 
 export type LogLevel = (typeof LOG_LEVELS)[number];
+
+// the name of repos
+export enum repoKeywords {
+  DeployInv = 'DeployInv'
+}
```

Comment:
Changing the enum in this fashion eliminates the need for the map
``` suggestion
// the name of repos
export enum repoKeywords {
  DeployInv = 'https://github.com/mingxuanzhangsfdx/DeployInv'
}
```

Reply:
we still need a string to construct the path